### ISSUE TITLE
harvest.log summary not correct

### ISSUE THREAD

**Issue by plawton-umd** [NASA-PDS/harvest#149]
### Checked for duplicates

Yes - I've already checked

### 🐛 Describe the bug

When I did ran havest and reviewed the log, I noticed the summary listed 269 files as skipped and 3 files as failed.

### 🕵️ Expected behavior

I expected correct information.

Reviewing the 269 skipped files - it was actually 150 unique files that were skipped one or multiple times.  The skips were because  the LIDVID/LID LID already exists in registry database.  In a few specific cases reviewed, the file was loaded within minutes before the skip.  However, they appear not to be included in the "Loaded_files" count.  They are accessible via an OpenSearch query.

Reviewing the 3 failed files - it was 2 unique files.  One had 2 different errors "sessions" - (it appears) the 2nd one happened although it had been loaded- although the first error session looks like it was not loaded.  Both files are accessible via an OpenSearch query.  If that is not how the log should be read, the error message needs to list the file that failed please.

### 📜 To Reproduce

1. I do not know how to force these errors.

### 🖥 Environment Info

- Version of this software 3.8.2
- Operating System: Linux

### 🩺 Test Data / Additional context
N/A

### 🦄 Related requirements
Tightly coupled with
* #148
* #147

### ⚙️ Engineering Details

N/A

**Comment by tloubrieu-jpl** [comment]
Hi @al-niessner ,

Could the fix made on https://github.com/NASA-PDS/registry-mgr/pull/116 also help here ?

**Comment by al-niessner** [comment]
@tloubrieu-jpl 

I do not think so. harvest tries to do all of it own counting and stuff. It is not overly robust and leaves a lot of room for double counting or miss counting. It does not query back to the database to see what is there. That is an idea though for more robust what actually happened.
